robottelo/host_helpers/satellite_mixins.py

```diff
@@ -107,6 +110,42 @@ def md5_by_url(self, url):
             f'wget -qO - {url} | tee {filename} | md5sum | awk \'{{print $1}}\''
         ).stdout
 
+    def upload_manifest(self, org_id, manifest=None, interface='API', timeout=None):
+        """Upload a manifest with locking, using the requested interface.
+
+        :type org_id: int
+        :type manifest: robottelo.manifests.Manifest
+        :type interface: str
+        :type timeout: int
+
+        :returns: the upload result
+
+        Note: The manifest uploading is strictly locked only when using this
+            function
+
+        """
+
+        if manifest is None:
+            pass
```

Comment:
I think we should change this to match the `upload_manifest_locked` pattern, then test to make sure it fails over properly in the event of a manifester failure:
```suggestion
            manifest = clone()
```
The easiest way to force a manifester failure is probably to change the value of the `SAT_VERSION` setting in `conf/manifest.yaml` to an invalid version. For example, `sat-6.13` as the value of that setting should cause a failure. @shweta83 let me know if you need any help setting up such a test.

Reply:
@synkd Updated.